module_3/gift_certificates_system/model/src/main/java/com/epam/esm/util/UrlMapping.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.esm.util;
+
+/**
+ * Url mapping for controllers.
+ */
+public final class UrlMapping {
+
+    /**
+     * General URLs.
+     */
+    public static final String ID = "/{id}";
+
+    /**
+     * URL for Gift certificate entities
+     */
+    public static final String GIFT_CERTIFICATES = "/gift-certificates";
+
+    /**
+     * URL for Tag
+     */
+    public static final String TAGS = "/tags";
+    public static final String MOST_WIDELY_USED_TAG_OF_USER_WITH_HIGHEST_COST_OF_ALL_ORDERS = "/most-widely";
+
+    /**
+     * URL only for User.
+     */
+    public static final String USERS = "/users";
+
+    /**
+     * URL for User with Order
+     */
+    public static final String ORDER_USER = "/{userId}/orders";
+
+    /**
+     * URL for Order
+     */
+    public static final String ORDERS = "/orders";
```

Comment:
Is there a need for orders to have a separate path? They seem to be dependent resource

Reply:
Early you said to transfer the method for creating orders to `OrderController`, so this mapping will be there.